spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -442,7 +442,9 @@ def make_a_string(item):
 def merge_dict(dict_in):
     """
     Merge n dictionaries that are contained at the root key
-    Example:
+
+    code::
+
```

Comment:
Not sure which you prefer, but if  `    .. code-block:: python` is used instead of `code::`, then the "code:" text will be omitted.

https://spinalcordtoolbox.com/en/aj-2951-fix_doc_warnings/dev_section/api.html#spinalcordtoolbox.aggregate_slicewise.merge_dict 

vs. 

![Screenshot from 2020-10-22 15-06-11](https://user-images.githubusercontent.com/16181459/96918355-2e477c80-1478-11eb-9835-2bb5e34cdabe.png)

Reply:
Yes much better